src/axom/primal/geometry/Vector.hpp

```diff
@@ -347,6 +357,18 @@ class Vector
   AXOM_HOST_DEVICE
   void negate();
 
+  /*!
+   * \brief In-place normalization of the vector.
+   */
+  AXOM_HOST_DEVICE
+  void normalize();
+
+  /*!
+   * \brief In-place unitization (normalization) of the vector.
+   */
+  AXOM_HOST_DEVICE
+  void unitize();
```

Comment:
Suggestion: Is there a good reason to have two functions that do the same thing? 
If not, my preference would be to only have `normalize()` (which is the conventional way to refer to this operation).